src/bundles/sound/functions.ts

```diff
@@ -391,43 +392,31 @@ export function play(sound: Sound): AudioPlayed {
     riffwave.header.bitsPerSample = 16;
     riffwave.Make(channel);
 
-    /*
-    const audio = new Audio(riffwave.dataURI);
-    const source2 = audioplayer.createMediaElementSource(audio);
-    source2.connect(audioplayer.destination);
-
-    // Connect data to output destination
-    isPlaying = true;
-    audio.play();
-    audio.onended = () => {
-      source2.disconnect(audioplayer.destination);
-      isPlaying = false;
-    }; */
-
     const soundToPlay = {
       toReplString: () => '<AudioPlayed>',
       dataUri: riffwave.dataURI,
     };
     audioPlayed.push(soundToPlay);
-    return soundToPlay;
+    return sound;

```

Comment:
Why was this REPL representation changed?

Reply:
Our "display" functions in modules are usually the identity function. I see the `play` functions to be in that category. The advantage is that you can debug your sounds by wrapping an expression that evaluates to a sound with a `play` call and it still will work.